pallets/automation-time/src/benchmarking.rs

```diff
@@ -45,6 +45,22 @@ fn schedule_tasks<T: Config>(owner: T::AccountId, time: u64, count: u32) -> T::H
 	task_id
 }
 
+fn schedule_transfer_tasks<T: Config>(owner: T::AccountId, time: u64, count: u32, recipient: T::AccountId, amount: BalanceOf<T>) -> T::Hash {
```

Comment:
no real need to have `recipient` and `amount` as params. Can just hardcode and say what they are in a fcn comment